great_expectations/expectations/metrics/column_map_metrics/column_value_lengths.py

```diff
@@ -1,6 +1,9 @@
 from typing import Optional
 
-import polars as pl
+try:
```

Comment:
🙇  Thank you! But let's use the import manager instead when it doesn't cause circular import issues (see https://github.com/great-expectations/great_expectations/pull/7100 for the fix).